.crossbar/config.yaml

```diff
@@ -52,11 +52,11 @@ workers:
     role: authenticator
 - id: coordinator
   type: guest
-  executable: python
+  executable: python3
   arguments:
   - -mlabgrid.remote.coordinator
   options:
-    workdir: ..
+    workdir: ../.crossbar
```

Comment:
This is basically the same as `workdir: .`.
I'm not really happy to change the generic config to make things a bit simpler for docker. @Emantor, what's your opinion?

Reply:
I would argue this is not only to make it simpler for docker but also making it more distinct what is configuration and what is files used for run-time persistence.

Reply:
You mean for the keys generated by crossbar? That's something I see as internal to crossbar, so keeping our persistence there as well seems strange. Maybe a separate directory?

Reply:
Ahh now I just read up on why I actually did this fix, sorry for the non precise answer above.
The ting is that we would like to persist the state of .crossbar in the docker host not i the container that's why we do the bind mount. 
However, the problem is that as off today the workdir for crossbar is /opt and not a distinct folder. (That's what this commit add) By running straight in /opt we have:
1) some confusion as it is not evident "who" uses these files 
2) The problem that e.g. also labgrid is installed there hence it can not be bind mounted from out-side. 
I tried bind mounting the relevant files in question instead of the complete folder. However, then I get a problem with your atomic write function as i tries to run a mv on the file, which you can not on a mounted file. That's why I thought that adding a specific .crossbar folder in /opt would be the best solution.